plugin/storage/integration/elasticsearch_test.go

```diff
@@ -156,9 +156,8 @@ func healthCheck() error {
 
 func testElasticsearchStorage(t *testing.T, allTagsAsFields bool) {
 	SkipUnlessEnv(t, "elasticsearch", "opensearch")
-	if err := healthCheck(); err != nil {
-		t.Fatal(err)
-	}
+	err := healthCheck()
+	require.NoError(t, err)
```

Comment:
```suggestion
	require.NoError(t, healthCheck())
```

Reply:
Fewer floating vars

Reply:
Don't know, it's more of a preference. Previous code had the err var scoped to the if statement, you pulled it to the function level, which in theory could cause side effects, so better to avoid that.